src/main/java/org/myrobotlab/service/Sabertooth.java

```diff
@@ -124,9 +122,10 @@ public void driveForwardMotor1(int speed) {
   public void driveForwardMotor2(int speed) {
     sendPacket(MOTOR2_FORWARD, speed);
   }
-
+  
   public void sendPacket(int command, int data) {
     try {
+      Serial serial = (Serial)getPeer("serial");

```

Comment:
could this introduce a performance bottleneck if we need to get and cast the serial port for every packet ?  could this handle be cached here or something instead of having to do it on each sendPacket request?

Reply:
Its 2 map lookups - and handing over a reference.   I'm testing it regularly - no indication of a performance problem